README.md

```diff
@@ -91,6 +92,14 @@ Your app is ready to be deployed!
 
 See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
 
+#### `yarn cache clean --force`
+
+Clear yarn cache.
+
+#### `yarn cache clean --force && rm -rf node_modules && yarn install`
+
+In order to be sure to correctly "clean" the app, this script is prefered from the one above.
```

Comment:
I don't understand here what script is preferred to which one. Plus, it seemed to me that clearing the cache was really important but can we add a little sentence to explain in what case(s) it needs to be done ?

Reply:
Just clean the cache may not always be suffcient to "clean" the webapp after package changes.
Sometimes, it also requires node_module directory remove then yarn install.